<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickGeocodingToolbox</class>
 <widget class="QDockWidget" name="QuickGeocodingToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick geocoding</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="cmbGeocoder"/>
    </item>
    <item>
     <widget class="QgsFilterLineEdit" name="txtSearch">
      <property name="toolTip">
       <string>Enter address for geocoding</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lstSearchResult"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
